### PR TITLE
Cf preview deploys

### PR DESCRIPTION
**Pull request by MTrost**
We couldn't get Traefik and Cloudflare to work together: X-Forwarded-Host header gets reset by Traefik. That's why this solution overwrites `location` and `base-uri` in request bodies.

This could break in the future (or breaks already on some Api calls).

We should look into that, but to get unstuck and preview deployments working we decided on going ahead with these changes.

### CODE REVIEW

File: .github/workflows/prev_deploy.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      contents: read
+      deployments: write
+      pull-requests: write
+    steps:
+      - name: Await CF Pages
+        uses: WalshyDev/cf-pages-await@v1
+        id: pages-action
+        with:
+          apiToken: ${{ secrets.CF_API_TOKEN }}
+          accountId: '9793bc27d30f5d738baf5e5a491b7416'
```
Comment (0xbase12) [inline .github/workflows/prev_deploy.yml:24]: there is the account id hardcoded and seems to be corresponding to you personal account on cloudflare. And we added the project under your sixsq email. should you update that value ?

Comment (MTrost) [reply .github/workflows/prev_deploy.yml:24]: uff yes, thanks, that was the problem 😄

File: .github/workflows/prev_deploy.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+name: Preview Deployment
```
[the remark this answers is not in the document]
Comment (MTrost) [reply .github/workflows/prev_deploy.yml:1]: The Cloudflare bot only shows you the most recent deployment.

This workflow adds a comment for every commit, you can see it here in my repository: https://github.com/MTrost/nuvla-ui/pull/2

I can imagine some scenarios where that could be helpful, when you work off feedback from the team or designer and can compare to previous versions. But it's definitely not essential and I have no problems removing the workflow to keep it cleaner.

Should I do that and remove it?

[the remark this answers is not in the document]
Comment (0xbase12) [reply .github/workflows/prev_deploy.yml:1]: Okey, more clear thanks. The essential info is latest build. But if we need to recover previous deployments urls, in the bot comment we can check the changes made by the Cloudflare bot:
Screenshot 2022-10-04 at 16 59 07

[the remark this answers is not in the document]
Comment (MTrost) [reply .github/workflows/prev_deploy.yml:1]: Ah nice, didn't know that, I remove the workflow!

**Comment by 0xbase12** [comment]
Can I delete the created secret?

**Comment by MTrost** [comment]
Yes you can, not needed anymore and I remove it from Cloudflare too.

**Pull request merged**